apiary.apib

```diff
@@ -5507,3 +5526,59 @@ and [buckets](https://keboola.docs.apiary.io/#reference/buckets/manage-bucket/bu
         {
             "displayName": "My-Display-Name"
         }
+
+# Group Direct Access
```

Comment:
Tohle už se chová jinak, ne? Neposílá to URL na heslo a má to endpoint na reset hesla.